toolkits/global/packages/global-forms/README.md

```diff
@@ -140,214 +213,256 @@ You can summarise errors using a top level `errorSummary` property. Each error i
-            "name": "search",
-            "id": "search",
-            "label": "Search",
-            "hideLabel": true
-        },
-        {
-            "template": "globalFormButton",
-            "type": "submit",
-            "label": "Search",
-            "modifiers": ["primary"]
-        }
-    ]
-}
+"options": [
+    {
+        "label": "Author affiliation",
+        "value": "University of Tokyo"
+    },
+    {
+        "label": "University of Tokyo",
+        "value": "University of Tokyo",
+        "selected": true
+    }
+]
 ```
 
-* **`fields`**: Individual fields are defined in a `fields` array.
-* **`gap`**: The gap between the fields; any valid `gap` value.
-* **`hideLabel`**: You can visually hide labels with the `hideLabel` property _without_ removing them from screen reader output. Only do this if there is another visual form of label (the “Search” label of the submit button in this case). 
-* **`nowrap`**: Not used in the previous example. Stop the individual fields from wrapping (beware of narrow viewports).
 
-### Supplementary fields
+## File upload
 
-You might want to show users an additional field when they select a particular radio or checkbox. For example, revealing a text input field for them to give more specific information about the option they’ve selected.
+![File upload](https://user-images.githubusercontent.com/15365576/210588295-d3e3119e-a3b0-44ca-be3a-8ee33416378b.png)
 
-These fields can have any properties of a standard form field. Set these properties using the `fields` property (an array), like this:
+### When to use this form element
 
-```json
-{
-    "label": "Monkey",
-    "value": "Monkey",
-    "name": "animal",
-    "fields": [
-        {
-            "template": "globalFormText",
-            "label": "What type of monkey?",
-            "id": "monkey-type",
-            "name": "monkey-type",
-            "hint": "Chimps are not monkeys"
-        }
-    ]
-}
-```
+File upload allows users to select and upload a file of their choosing. Use this element when it is a necessary part of a service e.g. an author manuscript as part of the publication process.
+
+### How it works
+
+The standard `accept` attribute is supported. Use this for filtering out file types that are not applicable for submission.
+
+
+## Buttons
+
+![Buttons](https://user-images.githubusercontent.com/15365576/210589049-ff61646f-c7c5-4dcf-9fc4-2b011b53ced0.png)
+
+### When to use this form element
+
+Use a button to let a user perform an action such as:
 
-### Buttons
+- Progressing to the next section of a form
+- Submitting a completed form
 
-A `template` of `buttons` defines a set of button controls, displayed inline (using Flexbox and `gap` for tidy wrapping). 
-The `type` property for each individual button corresponds to the `type` HTML attribute. For example, here is how you would include a submit button:
+Only use the primary button styling as the main call to action on a page.
```

Comment:
Could this be more like:

'Use the primary button styling for the main call to action on a page.' - otherwise it implies that on these pages you can only have the primary button and no others.

Reply:
```suggestion
Use the primary button styling for the main call to action on a page.
```